tests/phpunit/DecompressorZlibTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Tests\Processor\Decompress;
+
+use Keboola\Component\UserException;
+use Keboola\Processor\Decompress\Decompressor\DecompressorZlib;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+class DecompressorZlibTest extends TestCase
+{
+    public function testDecompressionReturnCodeDecompressionFail(): void
+    {
+        $this->expectException(UserException::class);
```

Comment:
expectException dej pred misto kde to ma failnout a pokud mozno tam prihod i expectExceptionMessage, takhle by to mohlo prochazet i pri user chybe v Ctoru